File: IDAUtil/Model/RejectionsDataProperty.vb
Imports System.ComponentModel.DataAnnotations.Schema

Public Class RejectionsDataProperty
    <Column("[salesOrg]")> Public Property salesOrg As String
    <Column("[country]")> Public Property country As String
    <Column("[soldTo]")> Public Property soldTo As Long
    <Column("[sku]")> Public Property sku As Long
    <Column("[skuDescription]")> Public Property skuDescription As String
    <Column("[skuUnitBarcode]")> Public Property skuUnitBarcode As String
    <Column("[skuCaseBarcode]")> Public Property skuCaseBarcode As String
    <Column("[rejectionReasonCode]")> Public Property rejectionReasonCode As String
    <Column("[needOutOfStockToReject]")> Public Property needOutOfStockToReject As Boolean
    <Column("[startDate]")> Public Property startDate As String
    <Column("[endDate]")> Public Property endDate As String
    <Column("[rejectionComment]")> Public Property rejectionComment As String
End Class

File: IDAUtil/Model/Properties/ServerProperty/RejectionsDataProperty.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace IDAUtil.Model.Properties.ServerProperty {
    public class RejectionsDataProperty {
        [Column("[salesOrg]")] public string salesOrg { get; set; }
        [Column("[country]")] public string country { get; set; }
        [Column("[soldTo]")] public int soldTo { get; set; }
        [Column("[shipTo]")] public int shipTo { get; set; }
        [Column("[sku]")] public int sku { get; set; }
        [Column("[skuDescription]")] public string skuDescription { get; set; }
        [Column("[skuUnitBarcode]")] public string skuUnitBarcode { get; set; }
        [Column("[skuCaseBarcode]")] public string skuCaseBarcode { get; set; }
        [Column("[rejectionReasonCode]")] public string rejectionReasonCode { get; set; }
        [Column("[needOutOfStockToReject]")] public bool needOutOfStockToReject { get; set; }
        [Column("[startDate]")] public DateTime startDate { get; set; }
        [Column("[endDate]")] public DateTime endDate { get; set; }
        [Column("[rejectionComment]")] public string rejectionComment { get; set; }
    }
}